llvm/test/Transforms/InstCombine/loadstore-metadata.ll

```diff
@@ -261,8 +261,8 @@ entry:
 define double @preserve_load_metadata_after_select_transform_metadata_missing_3(ptr %a, ptr %b) {
```

Comment:
Drop FIXME?

By the way, I think these tests would be clearer if you didn't have equal metadata on the first two loads, it's a bit unclear how exactly the preservation works that way.

Reply:
Dropped and added a new variant in 9e0ea8c8816d9f5837ad3357be32dddce28c7fb1

Reply:
made distinct....